homeassistant/components/tplink/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "TP-Link Smart Home",
   "codeowners": ["@rytilahti", "@bdraco", "@sdb9696"],
   "config_flow": true,
-  "dependencies": ["network"],
+  "dependencies": ["network", "ffmpeg", "stream"],
```

Comment:
It looks like we don't depend on `stream` and its actually `av`?

If thats the case we should add `av==13.1.0` to requirements instead of making `stream` a dep here.

Reply:
Its a bit messy though since `stream` loads `av`.....

Its the only place in the codebase we `import av` AFAICT.

Maybe we need to move `async_has_stream_auth_error` into stream and call it as `stream.async_check_stream` instead so we don't use `av` directly here.

Reply:
So a core change?

Reply:
This is now updated to use `stream.async_check_stream_client_error` from https://github.com/home-assistant/core/pull/132866.